src/java/org/apache/cassandra/repair/RepairRunnable.java

```diff
@@ -461,7 +462,14 @@ public ListenableFuture apply(List<RepairSessionResult> results)
                 // this node is implicitly a participant in this repair, so a single endpoint is ok here
                 if (!endpoints.isEmpty())
                 {
-                    filtered.add(new CommonRange(endpoints, transEndpoints, commonRange.ranges));
+                    Set<InetAddressAndPort> skippedReplicas = Sets.union(Sets.difference(commonRange.endpoints, endpoints),
+                                                                         Sets.difference(commonRange.transEndpoints, transEndpoints));
+                    skippedReplicas.forEach(endpoint -> logger.info("Removing a dead node {} from Repair for ranges {} due to -force", endpoint, commonRange.ranges));
```

Comment:
nit: Not entirely sure if we want DEBUG or INFO for these.

Reply:
I'd prefer to keep it as INFO. The message helps to reason about if some endpoints do not participate in a repair in a running server. `DEBUG` is typically disabled.